config/initializers/devise.rb

```diff
@@ -188,10 +193,10 @@
 
   # Number of authentication tries before locking an account if lock_strategy
   # is failed attempts.
-  # config.maximum_attempts = 20
+  config.maximum_attempts = 5
 
   # Time interval to unlock the account if :time is enabled as unlock_strategy.
-  # config.unlock_in = 1.hour
+  config.unlock_in = 1.hour
```

Comment:
maybe set config.unlock_strategy = :time implicit ?

Reply:
That is a good question. For now, I would offer email and time as unlocking strategy.This seems to be the default. Time seems a sensible option, because the main objective is to slow down BruteForce-attacks. E-Mail seems like a good alternative for those eager to quickly unlock things. If the email of a person is already compromised, hitobito cannot defend against wrong logins properly anyway.